utilities/AFQ_mrtrixInit.m

```diff
@@ -1,4 +1,4 @@
-function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder)
+function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder, mrtrixVersion)
```

Comment:
I haven't sorted through all this code but files are being written out with only the _dt.mif _sf.mif and so forth. They are being written out in the right location, but we are losing some aspect of the file name. @garikoitz can you check why that might be?